docs/data.json

```diff
@@ -1,119 +1,42 @@
-      "gpu_model": "* `NVIDIA Ampere GPU Architecture <https://www.nvidia.com/en-us/geforce/turing>`_\n* `Turing <https://www.nvidia.com/en-us/geforce/turing/>`_\n* `Volta <https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/>`_\n* `Pascal <https://www.nvidia.com/en-us/data-center/pascal-gpu-architecture/>`_",
-      "hugectr": "Not applicable",
-      "hugectr2onnx": "Not applicable",
-      "merlin.core": "0.3.0",
-      "merlin.models": "0.4.0",
-      "merlin.systems": "0.2.0",
-      "nvidia_driver": "NVIDIA Driver version 465.19.01\nor later is required.  However,\nif you're running on Data Center\nGPUs (formerly Tesla) such as T4,\nyou can use any of the following\nNVIDIA Driver versions:\n\n* 418.40 (or later R418)\n* 440.33 (or later R440)\n* 450.51 (or later R450)\n* 460.27 (or later R460)\n\n**Note**: The CUDA Driver\nCompatibility Package does not\nsupport all drivers.",
-      "nvidia_pytorch": "Not applicable",
-      "nvidia_tensorflow": "Not applicable",
-      "nvtabular": "1.1.1",
-      "openmpi": "4.1.2rc4",
-      "os": "Ubuntu 20.04.4 LTS",
-      "python_major": "3",
-      "pytorch": "1.11.0+cu113",
-      "release": "22.05",
-      "rmm": "21.12.0",
-      "size": "14.37 GB",
-      "sm": "Not applicable",
-      "sparse_operation_kit": "Not applicable",
-      "tensorrt": "8.2.4.2+cuda11.4.2.006",
-      "tf": "Not applicable",
-      "transformers4rec": "0.1.8",
-      "triton": "2.21.0"
-    }
-  },
   "nvcr.io/nvidia/merlin/merlin-hugectr": {
-    "22.05": {
-      "base_container": "Triton version 22.03",
-      "compressedSize": "5.54 GB",
-      "cublas": "11.8.1.74",
-      "cuda": "11.6.1.005",
-      "cudf": "22.2.0",
-      "cudnn": "8.3.3.40+cuda11.5",
-      "cufft": "10.7.1.112",
-      "curand": "10.2.9.55",
-      "cusolver": "11.3.3.112",
-      "cusparse": "11.7.2.112",
-      "cutensor": "1.5.0.1",
+    "22.06": {
+      "base_container": "Triton version 22.05",
+      "compressedSize": "6.57 GB",
+      "cublas": "11.10.1.25",
+      "cuda": "11.7.0.022",
+      "cudf": "22.4.0",
+      "cudnn": "8.4.0.27+cuda11.6",
+      "cufft": "10.7.2.50",
+      "curand": "10.2.10.50",
+      "cusolver": "11.3.5.50",
+      "cusparse": "11.7.3.50",
+      "cutensor": "1.5.0.3",
       "dgx_system": "* DGX-1\n* DGX-2\n* DGX A100\n* DGX Station",
       "gpu_model": "* `NVIDIA Ampere GPU Architecture <https://www.nvidia.com/en-us/geforce/turing>`_\n* `Turing <https://www.nvidia.com/en-us/geforce/turing/>`_\n* `Volta <https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/>`_\n* `Pascal <https://www.nvidia.com/en-us/data-center/pascal-gpu-architecture/>`_",
-      "hugectr": "Not applicable",
+      "hugectr": "3.7.0",
       "hugectr2onnx": "Not applicable",
-      "merlin.core": "0.3.0",
-      "merlin.models": "0.4.0",
-      "merlin.systems": "0.2.0",
+      "merlin.core": "0.4.0",
+      "merlin.models": "0.5.0+7.g886cf6de",
```

Comment:
I'm surprised these aren't plain old `0.5.0` and `0.3.0`, but I'm not saying anything is wrong here. I tend to believe these probably are the actual versions included in the containers, but maybe there were additional changes after the version tags to get the containers working?